snapshot/member_test.go

```diff
@@ -59,6 +59,9 @@ func TestSnapshotV3RestoreMultiMemberAdd(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// wait for membership reconfiguration apply
+	time.Sleep(testutil.RequestTimeout)
```

Comment:
apply should be fast. why we wait a request timeout here?

Reply:
> error validating peerURLs {ClusterID:2dcab7bbeafb497c Members:[&{ID:42fbaab60780e599 RaftAttributes:{PeerURLs:[unix://localhost:18469]} Attributes:{Name:0 ClientURLs:[unix://localhost:42777]}} &{ID:fd5b89a5f93ca727 RaftAttributes:{PeerURLs:[unix://localhost:29664]} Attributes:{Name:1 ClientURLs:[unix://localhost:35826]}} &{ID:336967b030dec3fa RaftAttributes:{PeerURLs:[unix://localhost:10625]} Attributes:{Name:2 ClientURLs:[unix://localhost:33018]}}] RemovedMemberIDs:[]}: member count is unequal

One of the peers still had 3 members after member-add command. Will change it to 1-second.

Reply:
probably add an apply timeout into test util.